steps/linux_steps.py

```diff
@@ -0,0 +1,381 @@
+#!/usr/bin/env python3
+# -*- coding: ascii -*-
+import logging
+
+from utils.ssh_client import SSHClient
+
+logger = logging.getLogger(__name__)
+
+
+class LinuxSteps(SSHClient):
+
+    def __init__(self,
+                 host=None,
+                 username=None,
+                 password=None,
+                 key=None,
+                 passphrase=None,
+                 add_system_known_hosts=False,
+                 **kwargs):
+
+        super().__init__(host, username, password, key, passphrase, **kwargs)
+        if add_system_known_hosts:
+            self.add_system_known_hosts()
+
+    @staticmethod
+    def signature_creator(command, options):
+        signature = [command]
+        for option in options:
+            if option is None or not options[option]:
+                continue
+            else:
+                signature.append(option)
+        return ' '.join(signature)
+
+    def step(self, signature, get_signature=False, **kwargs):
+        if get_signature:
+            return signature
+        else:
+            logger.info('Executing {}'.format(signature))
+            return self.execute(signature, **kwargs)
+
+    def script(self, *args, **kwargs):
+        return self.step(' '.join(args), **kwargs)
+
+    def cd(self,
+           path,
+           force_symbolic_links=False,
+           disregard_symbolic_links=False,
+           raise_error=False,
+           **kwargs):
+        options = {
+            '-L': force_symbolic_links,
+            '-P': disregard_symbolic_links,
+            '-e': raise_error,
+            path: True
+        }
+        return self.step(self.signature_creator('cd', options), **kwargs)
+
+    def mkdir(self,
```

Comment:
Remove this PROSTYNYA of parameters please :) Leave only main options, 3-5, 3 is better. And add parameter ```options``` to have ability specify any other options, e.g. ```options=['-l', '-a']```.

Reply:
Done